mapping-templates/Mutation.createOrganization.request.vtl

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Are we using VTL instead of, say, a Lambda Function? Will this be too limiting?

Reply:
With DynamoDb dataSources, vtl is the right choice for cost and performance (no cold start, no execution cost).